docs/Debug.md

```diff
@@ -1,57 +1,59 @@
-Debugging OpenNetVM
-==
+# Debugging OpenNetVM
```

Comment:
A few doc changes are fine to mix in with other PRs, but you have this + install doc which is about 400 line changes. Can you move it to another PR?

Reply:
Sorry I didn't see this, I'll fix it